app.py

```diff
@@ -9,7 +9,6 @@
     plot_box,
     plot_donut,
     plot_histo,
-    plot_line,
```

Comment:
@all4one-max Could you merge the main branch back to your branch in order to pull the latest code?

Reply:
Hey! I was trying to do that, but it didn't work out. so, I did hard reset, save my previous code, deleted my old branch, created the new branch and made the changes using the saved code.

btw for merge I was using the following command.
```
git checkout my-branch
git merge main
```
this was not working.

Reply:
Can you share the error/reason those commands are not working?

Reply:
There wasn't any error, everything executed find but the code didn't merge.